redesign/src/styles/layout/_404.scss

```diff
@@ -0,0 +1,129 @@
+.main.error-page {
+    margin-bottom: 160px;
+}
+
+.button {
+    &--option {
+        border: 1px solid $grey;
+        border-radius: 4px;
+        box-sizing: border-box;
+        cursor: pointer;
+        user-select: none;
+        @include padding (15px, 50px, 15px, 50px);
+        @include margin (15px, 30px, 25px, 30px);
+    }
+}
+
+a.button--option:hover {
+    background: $lightblack;
+    transition: background 0.7s ease-in-out;
+    color: #FFFFFF;
+    transition: color 0.7s ease-in-out;
+}
+
+.error.border {
```

Comment:
Same ampersand issue here; this could be included in the `.error` class below and incorporated like so :
```scss
&.border {
    margin-top: 40px;
}
```